Scripts/Python/xForcedViewCamera.py

```diff
@@ -0,0 +1,172 @@
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Additional permissions under GNU GPL version 3 section 7
+#
+# If you modify this Program, or any covered work, by linking or
+# combining it with any of RAD Game Tools Bink SDK, Autodesk 3ds Max SDK,
+# NVIDIA PhysX SDK, Microsoft DirectX SDK, OpenSSL library, Independent
+# JPEG Group JPEG library, Microsoft Windows Media SDK, or Apple QuickTime SDK
+# (or a modified version of those libraries),
+# containing parts covered by the terms of the Bink SDK EULA, 3ds Max EULA,
+# PhysX SDK EULA, DirectX SDK EULA, OpenSSL and SSLeay licenses, IJG
+# JPEG Library README, Windows Media SDK EULA, or QuickTime SDK EULA, the
+# licensors of this Program grant you additional
+# permission to convey the resulting work. Corresponding Source for a
+# non-source form of such a combination shall include the source code for
+# the parts of OpenSSL and IJG JPEG Library used as well as that of the covered
+# work.
+#
+# You can contact Cyan Worlds, Inc. by email legal@cyan.com
+#  or by snail mail at:
+#       Cyan Worlds, Inc.
+#       14617 N Newport Hwy
+#       Mead, WA   99021
+#
+# *==LICENSE==*/
+"""
+Module: xForcedViewCamera
+Age: global
+Date: November 23, 2025
+Author: Darryl Pogue
+Script to force a camera viewpoint from a clickable, with an optional vignette.
+"""
+
+from Plasma import *
+from PlasmaTypes import *
+from PlasmaKITypes import *
+import PlasmaControlKeys
+
+# define the attributes that will be entered in max
+Activator = ptAttribActivator(1, "Activation clickable")
+Camera = ptAttribSceneobject(2, "Viewpoint camera")
+Vignette = ptAttribString(3, "Optional vignette dialog - by Name", default=None)
+
+class xForcedViewCamera(ptModifier):
+    """
+    Modifier class to force a specific viewpoint camera upon clicking an
+    activator, with an optional GUI vignette dialog.
+
+    This is similar to how telescopes work, but without the multi-stage avatar
+    behaviour and without a player taking ownership of the clickable (i.e.,
+    multiple players can activate the viewpoint camera concurrently).
+    """
+
+    def __init__(self):
+        ptModifier.__init__(self)
+        self.id = 231
+        version = 1
+        minorVersion = 0
+        self.version = version
+        PtDebugPrint("__init__xForcedViewCamera v%d.%d" % (version, minorVersion))
+        self.inputInterface = ptInputInterface()
```

Comment:
This will run when 3ds Max inits. I'd prefer to see this happen later, eg in `OnInit()`